NetCoreRestApi/BusinessLayer/Managers/ProductManager.cs

```diff
@@ -4,16 +4,16 @@
 
 namespace BusinessLayer.Managers
 {
-    public class ProductManager : IProductManager
+    public class ProductManager<TId> : IProductManager<ProductModel<TId>>
```

Comment:
Remove TId from ProductManager class, it should inherit from generic BaseManager<TModel, TId> and IProductManager